s3config-test.yaml

```diff
@@ -98,9 +98,9 @@ S3_MOTR_CONFIG:                                     # Section for S3 Motr
    S3_MOTR_CASS_CLUSTER_EP: 10.10.1.3                 # Cassandra cluster end point
    S3_MOTR_CASS_KEYSPACE: "motr_index_keyspace"     # Cassandra keyspace
    S3_MOTR_CASS_MAX_COL_FAMILY_NUM: 1                 # Cassandra max column family number, default is 1
-   S3_UNIT_SIZES_FOR_MEMORY_POOL: [4096, 8192, 16384, 32768, 65536, 131072, 262144, 524288, 1048576, 2097152, 4194304]                          # Memory pool will be created for each of this unit_size with following properties
-   S3_MOTR_READ_POOL_INITIAL_BUFFER_COUNT: 10         # 10 blocks, the initial pool size, multiple of S3_MOTR_UNIT_SIZE
-   S3_MOTR_READ_POOL_EXPANDABLE_COUNT: 20             # 20 blocks, pool's expandable size, multiple of S3_MOTR_UNIT_SIZE
+   S3_UNIT_SIZES_FOR_MEMORY_POOL: [16384]              # Memory pool will be created for each of this unit_size with following properties
```

Comment:
Now we have single unit size, do we still need to show it as array ?

Also we can update comment to:
"Memory pool will be created for this size with following properties."

Reply:
Will address these changes in https://jts.seagate.com/browse/EOS-20630